src/logger.js

```diff
@@ -1,37 +1,52 @@
 /*
  * This file is part of ciboard-server
-
+ *
  * Copyright (c) 2021 Andrei Stepanov <astepano@redhat.com>
- * 
+ *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 3 of the License, or (at your option) any later version.
- * 
+ *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  * Lesser General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-const pino = require('pino');
 const util = require('util');
+
 const debug = require('debug');
+const pino = require('pino');
+const pinoPretty = require('pino-pretty');
+const pinoSentry = require('pino-sentry');
+
+/*
+ * Note: The Sentry DSN is read automatically from the environment variable
+ * SENTRY_DSN, as specified in the DeploymentConfig.
+ */
+const sentryStream = pinoSentry.createWriteStream();
+const pinoStreams = pino.multistream([
+  {
+    stream: pinoPretty(),
+    level: process.env.LOG_LEVEL || 'debug',
+  },
+  // Forward only warnings and errors to Sentry.
+  { stream: sentryStream, level: 'warn' },
+]);
 
 /**
  * off, fatal, error, warn, info, debug, trace
  */
-const logger = pino({ level: process.env.LOG_LEVEL || 'debug' });
+export const logger = pino.pino({}, pinoStreams);
```

Comment:
Could you please add info, for cases when Sentry is available, 
and Sentry is not available, for example in local-run.

Simple: if Sentry DSN is not configured, then it will log only to console.
if Sentry DSN is available then: ...

Reply:
Maybe add this to readme file too